orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -748,38 +746,40 @@ at::Tensor& add__Tensor(
   std::vector<OrtValue> ort_outputs_0_Mul(1);
 
   auto status = invoker.Invoke("Mul", {
-    std::move(ort_input_alpha),
-    std::move(ort_input_other),
-  }, ort_outputs_0_Mul, nullptr);
+                                          std::move(ort_input_alpha),
+                                          std::move(ort_input_other),
+                                      },
+                               ort_outputs_0_Mul, nullptr);
 
   if (!status.IsOK())
     throw std::runtime_error(
-      "ORT return failure status:" + status.ErrorMessage());
+        "ORT return failure status:" + status.ErrorMessage());
 
   auto ort_input_self = create_ort_value(invoker, self);
 
   std::vector<OrtValue> ort_outputs_1_Add(1);
   ort_outputs_1_Add[0] = ort_input_self;
 
   status = invoker.Invoke("Add", {
-    std::move(ort_input_self),
-    std::move(ort_outputs_0_Mul[0]),
-  }, ort_outputs_1_Add, nullptr);
+                                     std::move(ort_input_self),
```

Comment:
while it probably won't all fit on one line, I wonder about: (forgive the spacing...)
`status = invoker.Invoke(`
`  "Add",`
`  {std::move(ort_input_self), std::move(ort_outputs_0_Mul[0])},`
`  ort_outputs_1_Add,`
`  nullptr);`

Reply:
I think we could do:

```cpp
  status = invoker.Invoke("Add",
                          {std::move(ort_input_self), std::move(ort_outputs_0_Mul[0])},
                          ort_outputs_1_Add, nullptr);
```

(Note the trailing comma in the 2nd argument is removed...) It is just a little further off from similar code that is autogenerated.